Glyssen/BookScript.cs

```diff
@@ -190,6 +191,29 @@ public System.Collections.Generic.IReadOnlyList<Block> GetScriptBlocks(bool join
 			return list;
 		}
 
+		public List<Block> CombineBookChapterBlocks()
+		{
+			var list = new List<Block>(m_blockCount);
+
+			list.Add(m_blocks[0]);
+
+			for (var i = 1; i < m_blockCount; i++)
```

Comment:
Pretty sure the title block (if there is one) is always guaranteed to be the first block in the book, so this could be made much more efficient by just checking the first two blocks. We wouldn't even need to make a copy of the list if we weren't joining them.